Specification/8._Hardware_design_recommendations.md

```diff
@@ -221,16 +219,16 @@ The same principles apply to a triply-redundant bus as well.
 
 {% include lightbox.html url="/Specification/figures/can_chaining_doubly_redundant.png" title="Doubly-redundant CAN bus wiring" %}
 
-UAVCAN permits to keep one of the redundant interfaces unused.
+UAVCAN permits keeping one of the redundant interfaces unused.
 An example of such wiring is shown on the following figure.
 
 {% include lightbox.html url="/Specification/figures/can_chaining_doubly_redundant_one_unused.png" title="Non-redundant CAN bus wiring where nodes are equipped with redundant interfaces" %}
 
 ### Devices with different number of redundant interfaces
 
-Mission critical devices and non-mission critical devices often may need to co-exist on the same UAVCAN network.
+Mission critical devices and non-mission critical devices often need to co-exist on the same UAVCAN network.
```

Comment:
Can you check this paragaph still matches your intention :-)

Reply:
It does.

Reply:
Thanks.